spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Smoke test for demo app', function() {
+    
+    browser.waitForAngularEnabled(false);
+    browser.manage().window().maximize();
+    browser.get('http://automationpractice.com/index.php?controller=authentication&back=my-account');
+    
+
+it('Initial build testing', function() {
```

Comment:
you should format test case title as "Test case id: Test case title" from your test case document